soh/soh/OTRGlobals.cpp

```diff
@@ -323,6 +332,33 @@ OTRGlobals::OTRGlobals() {
 OTRGlobals::~OTRGlobals() {
 }
 
+void OTRGlobals::ScaleImGui() {
+    float scale = CVarGetFloat("gImGuiScale", defaultImGuiScale);
+    float newScale = scale / previousImGuiScale;
+    ImGui::GetStyle().ScaleAllSizes(newScale);
+    ImGui::GetIO().FontGlobalScale = scale;
+    previousImGuiScale = scale;
+}
+
+ImFont* OTRGlobals::defaultFontSize(float size) {
+    auto mImGuiIo = &ImGui::GetIO();
+    ImFontConfig font_cfg = ImFontConfig();
+    font_cfg.OversampleH = font_cfg.OversampleV = 1;
+    font_cfg.PixelSnapH = true;
+    font_cfg.SizePixels = size;
+    ImFont* font = mImGuiIo->Fonts->AddFontDefault(&font_cfg);
+    // FontAwesome fonts need to have their sizes reduced by 2.0f/3.0f in order to align correctly
+    float iconFontSize = size * 2.0f / 3.0f;
+    static const ImWchar sIconsRanges[] = { ICON_MIN_FA, ICON_MAX_16_FA, 0 };
+    ImFontConfig iconsConfig;
+    iconsConfig.MergeMode = true;
+    iconsConfig.PixelSnapH = true;
+    iconsConfig.GlyphMinAdvanceX = iconFontSize;
+    mImGuiIo->Fonts->AddFontFromMemoryCompressedBase85TTF(fontawesome_compressed_data_base85, iconFontSize,
+                                                          &iconsConfig, sIconsRanges);
+    return font;
+}
+
```

Comment:
Not a huge fan of the naming here. For one thing, the casing is off, first letter should be capital like every other function. Second, it sounds like a property as if `defaultFontSize` is some static value, but what it looks like this is doing is adding a new font with that size. So it should probably have a name like, `ConstructDefaultFontWithSize` or something along those lines.

Reply:
Makes sense. I'll modify it when I get home.